packages/grafana-data/src/events/common.ts

```diff
@@ -1,20 +1,23 @@
 import { DataFrame } from '../types';
 import { BusEventWithPayload } from './types';
 
-/** @alpha */
+/**
+ * When hovering over an element this will identify
+ *
+ * For performance reasons, this object will usually be mutated between updates.  This
+ * will avoid creating new objects for events that fire frequently (ie each mouse pixel)
+ *
+ * @alpha
+ */
 export interface DataHoverPayload {
-  raw: any; // Original mouse event (includes pageX etc)
-
-  x: Record<string, any>; // { time: 5678 },
-  y: Record<string, any>; // { __fixed: 123, lengthft: 456 }  // each axis|scale gets a value
-
   data?: DataFrame; // source data
   rowIndex?: number; // the hover row
   columnIndex?: number; // the hover column
   dataId?: string; // identifying string to correlate data between publishers and subscribers
 
-  // When dragging, this will capture the original state
-  down?: Omit<DataHoverPayload, 'down'>;
+  // When dragging, this will capture the point when the mouse was down
+  point: Record<string, any>; // { time: 5678, lengthft: 456 }  // each axis|scale gets a value
```

Comment:
supernit: Does it need to be `any` in the interface? According to the comment, it should be a number, right?

Reply:
could be anything really -- for a timeseries graph that only supports numbers it will be numbers, but the main event should support anything hovered at that point.